htdocs/projet/contact.php

```diff
@@ -105,7 +105,7 @@
 			foreach ($task_array as $task) {
 				$task_already_affected = false;
 				$personsLinked = $task->liste_contact(-1, $source);
-				if (!is_array($personsLinked) && count($personsLinked) < 0) {
+				if (!is_array($personsLinked) || count($personsLinked) == 0) {
```

Comment:
I think what we wanted to do is:
If not array && < 0
Not array test is to avoid to compare array with int and < 0 is to detect if liste_,contact was in error.
Empty array is not an error case because object can have no contact at all.

Reply:
OK, I'll remove count() then.

Reply:
@eldy I modified the code.